pkg/config/constants.go

```diff
@@ -4,6 +4,6 @@ const (
 	UserConfigDirectoryNotFoundErrorMessage = "User config directory not found"
 	ObsidianConfigDirectory                 = "obsidian"
 	ObsidianConfigFile                      = "obsidian.json"
-	ObsidianCLIConfigDirectory              = "obs"
```

Comment:
Config dir needs to be migrated, I'm not sure what is a process for that. Maybe we can at least migrate in for aur package? For the very least this need to be in release notes!

`mv ~/.config/obs ~/.config/obsidian-cli` I assume, I'm yet to use this tool :D

Reply:
We should leave the config directory as it is, unless `obs` in the file path is already being used by OBS studio which I don't believe it is, we don't need to do that. Its not shown to the user at all so it won't affect their experience or consistency.

Reply:
I think it is a just a right moment to change it so we don't run into similar issue in a future. For someone like me who manage their dotfiles it would defiantly be inconsistent because many people strongly associate "obs" with "OBS Studio" and not "Obsidian". That's a reason I got involved with this issue in a first place.

Reply:
Fair enough its a good time to change but I feel cautious to change it without proper testing, as I am not using Obsidian much right now and busy with other things, it would be hard for me to ensure that everything is working, especially for users who are already using this and would migrate their configs. I was hoping you were testing as you made these changes but you said you haven't used this tool yet.

Reply:
Yeah I didn't wanted to move my obs studio binary, I did anyway to test now. The only thing that is stored in config is `default_vault_name` so even if old config is lost forever user would just have to run `obsidian-cli set-default <vault name>` once more after they get "Cannot find vault config, please use set-default command to set default vault or use --vault flag" error.
But you can just `mv ~/.config/obs ~/.config/obsidian-cli` which I did.
I would've suggested adding legacy path for old config location and give user a notice to migrate their config but since it contains single entry I don't think it's that big of a deal.

Reply:
Sure let's proceed with the change then.